### PR TITLE
Update CodeOwners

### PR DESCRIPTION
**Pull request by mrunfeldt**
* Removed Kevin and Gera
* Added Chris, Mel, Winters, Michael

### CODE REVIEW

**Review by tovbinm — APPROVED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -5,4 +5,4 @@
 # These owners will be the default owners for everything in the repo.
 # Unless a later match takes precedence, these users will be requested
 # for review when someone opens a pull request.
-* @leahmcguire @tovbinm @wsuchy @Jauntbox @gerashegalov @nicodv
+* @leahmcguire @tovbinm @wsuchy @mrunfeldt @crupley @nicodv @michaelweilsalesforce @WintersLu
```
Comment (tovbinm) [inline .github/CODEOWNERS:8]: is `@wsuchy` onboard or can be removed?

Comment (wsuchy) [reply .github/CODEOWNERS:8]: Indeed, you can remove me as I no longer work at Salesforce. 
BTW Could someone add me to the list of contributors?

Comment (tovbinm) [reply .github/CODEOWNERS:8]: @wsuchy added.

Comment (mrunfeldt) [reply .github/CODEOWNERS:8]: Aw, I miss you guys.

**Pull request merged**